### PR TITLE
Hack: Get chalice to generate resonable permissions

### PR DESCRIPTION
**Pull request by mattiaslundberg**
https://github.com/aws/chalice/issues/692

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -36,12 +37,24 @@
...
+    sqs.get_queue_url()
+
+
 @app.route(
     "/interactive",
     methods=["POST"],
     content_types=["application/x-www-form-urlencoded"],
 )
 def interactive():
+    if False:
```
Comment (parberge) [inline app.py:56]: Why have this condition. When won't it be `False`?

**Review by parberge — COMMENTED** [review]
Do you see this as a solution until chalice does the fix?
Or is it a solution until we solve it another way regardless of if chalice fixes the issue or not?

File: app.py (hunk 1 above)
Comment (mattiaslundberg) [reply app.py:56]: To get chalice to parse the function...

**Comment by mattiaslundberg** [comment]
Until we or chalice have a better fix for this. But likely it will be for us to manually generate the policy we need

**Review by parberge — APPROVED** [review]
I really think we should just handle it manually, but I guess it will do fine for now

**Pull request merged**